glean-core/src/lib.rs

```diff
@@ -499,7 +499,7 @@ impl Glean {
         }
 
         self.upload_manager
-            .process_ping_upload_response(uuid, status);
+            .process_ping_upload_response(uuid, status, self.debug_view_tag());
```

Comment:
Why was it added in this PR?

Reply:
Because when I changed the `process_ping_upload_response` function to use `enqueue_ping` instead of just adding a request to the queue I realized it was not taking the debug tag in consideration.